### PR TITLE
Validator Controller

### PR DESCRIPTION
**Pull request by GabrielMartinezRodriguez**
## Description

- Add a pallet for manage the validators after the genesis
- Add a EVM precompile for interact with the validator-set-pallet from the evm
- Add pallet session for managing sessions
- Add pallet im-online for remove offline validators
- Created stability-cli with the first command: `start-validation` which helps to configure a validator node

## Types of changes

What types of changes does your code introduce?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by clostao — DISMISSED** [review]
LGTM

**Review by deblanco — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -6,8 +6,10 @@ members = [
     'node',
     'runtime',
     'precompiles/balances-erc20',
+    'precompiles/validator-controller',
     'precompiles/utils',
-	'pallets/root-controller'
+	'pallets/root-controller',
```
[the remark this answers is not in the document]
Comment (GabrielMartinezRodriguez) [reply Cargo.toml]: solve

File: precompiles/validator-controller/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+sha3 = "0.10"
+sp-io = { workspace = true }
+
+[features]
+default = [ "std" ]
+std = [
+    "precompile-utils/std",
+    "sp-core/std",
+	"fp-evm/std",
+	"precompile-utils/std",
```
[the remark this answers is not in the document]
Comment (GabrielMartinezRodriguez) [reply precompiles/validator-controller/Cargo.toml]: solve

**Review by clostao — APPROVED** [review]
LGTM! Good job Gabriel!

**Pull request merged**